cmd/kubeadm/app/phases/addons/proxy/proxy_test.go

```diff
@@ -114,6 +115,14 @@ func TestEnsureProxyAddon(t *testing.T) {
 		},
 	}
 
+	// Override the default timeouts to be shorter
+	defaultTimeouts := kubeadmapi.GetActiveTimeouts()
+	defaultAPICallTimeout := defaultTimeouts.KubernetesAPICall
+	defaultTimeouts.KubernetesAPICall = &metav1.Duration{Duration: time.Microsecond * 500}
+	defer func() {
+		defaultTimeouts.KubernetesAPICall = defaultAPICallTimeout
+	}()
+
```

Comment:
doing these overrides for tests seems OK,
i ran `time go test ...` and saw no major increase in time for our unit tests.